rws/rws-be/src/test/java/my/RouteControllerTestIT.java

```diff
@@ -0,0 +1,168 @@
+package my;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import entity.Route;
+import entity.WayPoint;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.IntegrationTest;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.TestRestTemplate;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.web.client.RestTemplate;
+import ua.vertex.config.DBConfig;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Created by user on 25.06.2016.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = {Application.class, DBConfig.class})
+@ActiveProfiles("test")
+@WebAppConfiguration
+@IntegrationTest({"server.port=8081"})
+
+public class RouteControllerTestIT {
+
+
+    private final String baseURL = "http://localhost:8081/route";
+
+
+    @Test
+    public void testGetRoute() throws Exception {
+        RestTemplate template = new TestRestTemplate();
+        ResponseEntity<String> response = template.getForEntity(baseURL + "/1", String.class);
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode responseJson = objectMapper.readTree(response.getBody());
+        JsonNode messageJson = responseJson.path("name");
+        assertThat(messageJson.asText(), equalTo("first"));
+
+
+    }
+
+    @Test
+    public void testAddRoute() throws Exception {
+        RestTemplate rest = new TestRestTemplate();
+        Route route = new Route();
+        route.setId(4);
+        route.setName("name4");
+        List<WayPoint> waypoints = new ArrayList<>();
+
+        WayPoint.Builder builder = WayPoint.newBuilder();
```

Comment:
It's a builder, use form like this: builder.set(..)
                                                  .set(...)
                                                  .set(...);